introduction.md

```diff
@@ -1,2 +1,5 @@
 aaa bbbb ccccc dddd eee fff ggg  hhhh iii jjj kkk lll mmm nnn ooo pp qqq rrr ss ttttt uuu vvv wwww xx yyyy zzz
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?